docs/src/_guide/integration.md

```diff
@@ -37,12 +37,12 @@ On a shell, you can explicitly `export` environment variables, that will be pass
 ```bash
 export TAP_FOO_BAR=bar
 export TAP_FOO_BAZ=baz
-meltano elt ...
+meltano run ...
 
-TAP_FOO_BAR=bar TAP_FOO_BAZ=baz meltano elt ...
+TAP_FOO_BAR=bar TAP_FOO_BAZ=baz meltano run ...
 ```
 
-To verify that these environment variables will be picked up by Meltano as you intended, you can test them with [`meltano config <plugin>`](/reference/command-line-interface#config) before running `meltano elt`.
+To verify that these environment variables will be picked up by Meltano as you intended, you can test them with [`meltano config <plugin>`](/reference/command-line-interface#config) before running `meltano run`.
 
 If you're using [`meltano schedule`](/reference/command-line-interface#schedule) to [schedule your pipelines](/guide/orchestration), you can specify environment variables for each pipeline in your [`meltano.yml` project file](/concepts/project#meltano-yml-project-file), where each entry in the `schedules` array can have an `env` dictionary:
 
```

Comment:
@sbalnojan just below this is the non-job format for schedules, would you want to update that as well? The current format uses `elt` behind the scenes while the jobs format would use `run`.